tests/integration/kubernetes/k8s-policy-pod.bats

```diff
@@ -63,10 +63,8 @@ wait_for_pod_ready() {
 }
 
 @test "Able to read env variables sourced from configmap using envFrom" {
-	kubectl create -f "${correct_configmap_yaml}"
-	kubectl create -f "${correct_pod_yaml}"
-	kubectl wait --for=condition=Ready "--timeout=${timeout}" pod "${pod_name}"
-	expected_env_var=$(kubectl exec "${pod_name}" -- printenv data-3)
+	wait_for_pod_ready
+	expected_env_var=$(kubectl exec "${pod_name}" -- ${exec_command[@]})
```

Comment:
```suggestion
	expected_env_var=$(kubectl exec "${pod_name}" -- "${exec_command[@]}")
```
I think this needs to be quoted to prevent word splitting of the `exec_command` array entries.

Reply:
Good point, but I would like to clean that up in a future PR, because:
- We don't seem to have a splitting problem in this particular example, and
- Running CI tests again after they passed is a relatively expensive activity.